setup.py

```diff
@@ -21,8 +21,6 @@ def main():
         # not zip-safe.
         zip_safe=False,
         cmdclass=versioneer.get_cmdclass(),
-        tests_require=['six'],
-        test_suite='conda_smithy',
```

Comment:
Would it be possible to do something like [this]( http://pytest.org/2.2.4/goodpractises.html#integration-with-setuptools-distribute-test-commands )?